eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/entity/client/ClientEntity.java

```diff
@@ -50,6 +49,10 @@ public class ClientEntity extends BaseEntity {
     @Schema(name = "status", defaultValue = "0", allowableValues = {"0", "1"}, description = "0:not active, 1:active")
     private Integer status;
 
+    private String configIds;
+
     private String description;
+
+    private Timestamp endTime;
 }
```

Comment:
Is this field necessary?

Reply:
Endtime is the time client terminatted by exception, or signal.
If the exact time is difficult to obtain, it can be the time client termination is detected.